### DISCUSSION TITLE
Add ability to toggle between light and dark mode for Control Preview Window

### DISCUSSION THREAD

**Discussion by AndyHTML2012** [nptr/msstyleEditor#110] [Ideas]
It would be nice to be able to toggle between light and dark mode for win32 dark mode in the control preview window to see what those controls look like. It's hard to tell what most of the dark mode visuals look like without seeing them in live preview.

**Comment by nptr** [comment]
Ufortunately its already quite hard to make an application to use dark mode, let alone changing it on the fly. You'll have to bug Microsoft to fix their shit and have them provide a proper API for that. Last time I checked, to get dark mode on an application one would have to resort to calling lots of undocumented Windows functionality. Sorry, you'll likely have to wait for Windows 13 if ever, or for me to get bored enought to hack around the mess they have as user interfaces 💀
